skills/insforge-cli/references/db-migrations.md

```
@@ -0,0 +1,201 @@
+npx @insforge/cli db migrations up --to <migration-file-name-or-version>
+npx @insforge/cli db migrations up --all
+```
+
+## What Each Command Does
+
+| Command | Description |
+|--------|-------------|
+| `list` | Show applied remote migrations (version, name, created date) |
+| `fetch` | Download remote applied migrations into `migrations/` |
+| `new <migration-name>` | Create the next local migration file with the next timestamp version |
+| `up <filename\\|version>` | Apply exactly one explicit local migration file |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)